core/templates/dev/head/pages/collection-player-page/collection-player-page.directive.html

```diff
@@ -47,9 +47,8 @@ <h2 ng-if="$ctrl.collectionPlaythrough.hasFinishedCollection()" class="oppia-pag
       <a ng-repeat="node in $ctrl.collection.getCollectionNodes()"
          ng-href="<[$ctrl.getExplorationUrl(node.getExplorationId())]>"
          ng-style="{position: 'absolute', left: '<[$ctrl.pathIconParameters[$index].left]>', top: '<[$ctrl.pathIconParameters[$index].top]>'}">
-
         <div ng-style="{width: '160px',position: 'absolute',left: '<[$ctrl.getExplorationTitlePosition($index)]>',color: '#006553','font-family': 'Capriola, Roboto, Arial,sans-serif','font-size': '18px',bottom: '65%'}">
-          {{node._explorationSummaryObject.title}}
+          <[node.getExplorationTitle()]>
```

Comment:
Why this change?

Reply:
I think this was an unexpected change merged in PR #6881.
1. We use `<[]>`for angularJS interpolation.
2. Accessing private property `_explorationSummaryObject` dierectly!

/cc @Showtim3

Reply:
Ok, thanks for the explanation!